dstu3/src/main/java/org/opencds/cqf/dstu3/providers/MeasureOperationsProvider.java

```diff
@@ -555,6 +555,10 @@ public Parameters report(@OperationParam(name = "periodStart", min = 1, max = 1)
         Date periodStartDate = DateHelper.resolveRequestDate(periodStart, true);
         Date periodEndDate = DateHelper.resolveRequestDate(periodEnd, false);
       
+        if (periodStartDate.after(periodEndDate)) {
```

Comment:
Are these guaranteed not null by the resolveRequestDate call? If not, suggest a check